testing-common/testing-common.gradle

```diff
@@ -67,6 +67,8 @@ tasks.register("testDisabledFieldInjection", Test) {
   includes = ["context/FieldBackedProviderFieldInjectionDisabledTest.class"]
 }
 test.dependsOn(testDisabledFieldInjection)
-test.forkEvery 1
+tasks.withType(Test) {
```

Comment:
Noticed that since this approach doesn't override test tasks like the previous one did, some configs get more complicated. I think it's rare so it might be ok but I can go back to the approach of overriding test instead of defining new tasks if it makes more sense.

Reply:
Found another file that was affected and realized it's probably hard to reason about right now, went back to the approach of using a rule.